app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -252,6 +251,9 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
         }
     }
 
+    // schedule task to save the watch position each second
+    private var watchPositionTimer = Timer()
```

Comment:
```suggestion
    private var watchPositionTimer: Timer? = null
```
We don't need to initialize the timer here - it's going to be overwritten before the first usage anyways.